examples/simple-filter-xml-mqtt/main.go

```diff
@@ -0,0 +1,79 @@
+//
+// Copyright (c) 2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/edgexfoundry/app-functions-sdk-go/pkg/edgexsdk"
+	"github.com/edgexfoundry/app-functions-sdk-go/pkg/excontext"
+	"github.com/edgexfoundry/go-mod-core-contracts/models"
+)
+
+const (
+	serviceKey = "sampleFilterXml"
+)
+
+func main() {
+	// 1) First thing to do is to create an instance of the EdgeX SDK and initialize it.
+	edgexSdk := &edgexsdk.AppFunctionsSDK{ServiceKey: serviceKey}
+	if err := edgexSdk.Initialize(); err != nil {
```

Comment:
I thought you removed Initialize and create a New function??

Reply:
Well i was on the fence about it, I can make that change in a separate pull request.

Reply:
Oh, I thought you had already done that. I like the NewXYZ() pattern. Yes a separate PR is appropriate.